docs/admin/templates/extending-templates/devcontainers.md

```diff
@@ -3,6 +3,36 @@
 To enable dev containers in workspaces, configure your template with the dev containers
 modules and configurations outlined in this doc.
 
+> [!NOTE]
+>
+> Dev containers require a **Linux or macOS workspace**. Windows is not supported.
+
+## Configuration Modes
+
+There are two approaches to configuring dev containers in Coder:
+
+### Manual Configuration
+
+Use the `coder_devcontainer` Terraform resource to explicitly define which dev
+containers should be started in your workspace. This approach provides:
+
+- Predictable behavior and explicit control
+- Clear template configuration
+- Easier troubleshooting
+- Better for production environments
+
+This is the recommended approach for most use cases.
+
+### Project Discovery
+
+Enable automatic discovery of dev containers in git repositories using
+environment variables. This approach:
+
+- Automatically scans for `.devcontainer/devcontainer.json` files
```

Comment:
Only this specific path? IIRC `[root]/devcontainer.json` and some other permutations are also valid.